pkg/proxy/userspace/roundrobin_test.go

```diff
@@ -97,6 +74,40 @@ func expectEndpointWithSessionAffinityReset(t *testing.T, loadBalancer *LoadBala
 	}
 }
 
+func TestBuildPortsToEndpointsMap(t *testing.T) {
```

Comment:
likewise. This isn't testing anything in `pkg/proxy/userspace`, it should go in `pkg/proxy/utils`